python/cuml/cuml/metrics/cluster/entropy.pyx

```diff
@@ -27,22 +22,6 @@ cdef extern from "cuml/metrics/metrics.hpp" namespace "ML::Metrics" nogil:
                    const int upper_class_range) except +
 
 
-@cuml.internals.api_return_generic()
-def _prepare_cluster_input(cluster) -> typing.Tuple[CumlArray, int, int, int]:
```

Comment:
This didn't need to be split into a helper function (nor decorated). Just merged together while simplifying and migrating decorators.